wbooks/src/config/api.ts

```diff
@@ -0,0 +1,9 @@
+import { create } from 'apisauce';
+import Config from 'react-native-config';
+
+const api = create({
+  baseURL: Config.API_BASE_URL,
+  timeout: 5000
+});
+
```

Comment:
You forgot this Acceptance Criteria: 
![image](https://user-images.githubusercontent.com/8583214/106530145-87d53380-64ca-11eb-95e8-a75c19a4fcde.png)

Reply:
https://github.com/infinitered/reactotron-apisauce#configuring